scalableminds/webknossos
# Change GET routes with side effects to POST

Changing wk routes from GET to POST that have side effects, as described in #5988

Also removed unused route `checkInbox` (without the `Blocking`)

Note for reviewer: On the wk backend side I renamed the method `postWithJsonResponse` to `postJsonWithJsonResponse` to indicate that is indeed also sending data in the body. The new method `postWithJsonResponse` does not send data in the body.

### Steps to test:
- Create organization, directory should exist
- trigger a job,s hould be in the list
- reload datasets list in dashboard
- reload layer data, (“clear dataset cache”) should be refreshed 

### Issues:
- fixes #5988

------
